<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeSnippetView</class>
 <widget class="QWidget" name="CodeSnippetView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1261</width>
     <height>801</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="m">
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QLineEdit" name="nameEdit">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="typePicker">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="descriptionEdit">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QTabWidget" name="sourceSwitch">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="header">
       <attribute name="title">
        <string>Header</string>
       </attribute>
       <widget class="CodeEditor" name="codeEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>941</width>
          <height>671</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="source">
       <attribute name="title">
        <string>Source</string>
       </attribute>
       <widget class="CodeEditor" name="codeEditSrc">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>941</width>
          <height>671</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
